portfolio/src/main/java/com/google/sps/data/CommentsStore.java

```diff
@@ -41,4 +41,19 @@ public static void save(Comment comment) {
   public static List<Comment> load(int limit) {
     return ofy().load().type(Comment.class).order("-date").limit(limit).list();
   }
+
+  /**
+   * Loads and returns comments in the database sorted by date starting from the
+   * offset (included) and limited in number.
+   */
+  public static List<Comment> load(int limit, int offset) {
+    return ofy().load().type(Comment.class).order("-date").offset(offset).limit(limit).list();
```

Comment:
By now you've probably realized there is lot of repetition between methods around here. It might be a good moment to extract common part to a new (private) method.
However! There is convention around default parameters and method overloading. In short, methods with fewer number of parameters should call their more-parametrized counterparts, like this:
```
public X load(int limit) {
  return load(limit, 0);
}
```
This way you are not repeating code and keep most logic in one place (the one that has most parameters to customize behavior).

Reply:
I wasn't aware of that convention, good to know!